tools/jtest/jtest.cc

```diff
@@ -3443,19 +3443,16 @@ seen_it(char *url)
   if (!url_hash_entries) {
     return 0;
   }
-  union {
-    unsigned char md5[16];
-    uint64_t i[2];
-  } u;
   int l      = 0;
   char *para = strrchr(url, '#');
   if (para) {
     l = para - url;
   } else {
     l = strlen(url);
   }
-  ink_code_md5((unsigned char *)url, l, u.md5);
-  uint64_t x = u.i[0] + u.i[1];
+  CryptoHash hash;
+  CryptoContext().hash_immediate(hash, reinterpret_cast<void *>(url), l);
+  uint64_t x = hash.u64[0] + hash.u64[1];
```

Comment:
Shouldn't this be `uint64_t x = hash.fold();` in case other hashes want to fold differently?